compiler/rustc_data_structures/src/sorted_map/index_map.rs

```diff
@@ -152,6 +152,3 @@ impl<I: Idx, K, V> std::ops::Index<I> for SortedIndexMultiMap<I, K, V> {
         &self.items[idx].1
     }
 }
-
-#[cfg(tests)]
-mod tests;
```

Comment:
I think this should actually be left in but with `#[cfg(tests)]` changed to `#[cfg(test)]` right? (`compiler/rustc_data_structures/src/sorted_map/tests.rs` is a valid file)